### PR TITLE
Portfoliomodalpopup

### PR DESCRIPTION
**Pull request by stevenmukama**
Added the portfolio modal popup 
Added  the CSS styles for styling the page

### CODE REVIEW

**Review by anagudelogu — COMMENTED** [review]
Hi @stevenmukama, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the correct linter setup, please consider following steps by step this [guide](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+//   dynamic.innerHTML = `<div id="cards${i}" class="boxes">
+//   <div class="box-content">
+//   <h2>${namesarray[i]}</h2>
+//   <p>  ${descriptionarray[i]} </p>
+//   <p> <a class="showmore" href="#">${readmore[0]}</a></p>
+//   <img ${bgimg} />
+//   </div>
+//   </div>${(fetch)}`;
+//   bgimg.style.backgroundImage = `url('./images/${namesarray[i]}.png)`;
+// }
```
Comment (franclobo) [inline main.js:19]: - [ ] **[OPTIONAL]** Please 🙏🏽 remove the commented code to have a clean code that looks professional.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+        <div class="project-links">
+          <a href="" id="project-link-1" class="project-link-source">
+           See Source
+          <img src="/images/github.svg" alt="see-project-icon">
+        </a>
+        <a href="" id="project-link-1" class="project-link-live">
+        See live
+       <img src="/images/live.svg" alt="see-project-icon">
+     </a>
+        </div>
```
Comment (franclobo) [inline main.js:199]: - [ ] Please 🙏🏽 correct the path of the images, we could add a dot at the beginning `./images...` for a better user interface.

image

File: main.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    title: 'Website Protfolio',
+    popupdescription: 'Multi Post Stories',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required. has been the industrys standard',
+    objectdescription: 'Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    languages: ['html', 'Bootstrap', 'Ruby on rails'],
+    // image: 'SVG/w6bgdt.png',
+    livelink: 'https://stevenmukama.github.io/Microverseportfolio/',
+    sourceLink: 'https://github.com/stevenmukama/Microverseportfolio',
+  },
```
Comment (franclobo) [inline main.js:131]: - [ ] There are seven cards with the button `See project` you should have seven pop-up windows. Please 🙏🏽 add the pop-up window for a better user experience.

image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1769 @@
...
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .modal-content {
+    margin: 0;
+    padding: 30px;
+  }
```
Comment (franclobo) [inline style.css:1628]: - [ ] For the desktop version of the pop-up window it is too close to the right border, kindly center the pop-up window to look similar to the Figma template.
- [ ] For the mobile version the pop-up window is too close to the borders, please 🙏🏽 center it to look similar to the Figma template.

| Figma template | Your design |
|---|---|
| image | image |
| image | image |

File: README copy.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## Built With
+
+- html
+- css
+
+
+## Getting Started
+
+To get a local copy up and run git clone git clone https://github.com/stevenmukama/portfoliomobile.git and follow these simple steps
+
```
Comment (franclobo) [inline README copy.md:13]: - [ ] Kindly rename the file to `README.md` instead of `README copy.md` to do it possible to visualize on the repository. 👀 📑

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @stevenmukama, 

# Required Changes ♻️

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] ⚠️ Kindly review the announcements in the Slack channel they informed us about a change in the [README template](https://github.com/microverseinc/readme-template) IT IS GOING TO BE MANDATORY NEXT WEEK ⚠️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @stevenmukama, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions
[OPTIIONAL] Kindly make sure that your mobile version is showing all the pop-up content without horizontal and vertical scrolling. It helps to improve the user's experience. :+1: :smiley: :smiley: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**